components/03-modules/form/form--compact.hbs

```diff
@@ -0,0 +1,9 @@
+<form class="form-compact {{ class }}">
```

Comment:
```scss
.form-compact {
  .form {
    &__input { }
    &__label { }
  }
}
```
Then in markup you can easily change
```html
<form class="form">
```
to
```
<form class="form form--compact">
```
and all necessary styling will be aplied, without chaning classes of all child elements.